content/800-data-platform/05-data-browser.mdx

```diff
@@ -154,18 +164,17 @@ Use the **Filters** menu to filter data in the model by adding conditions.
       - **gt**
       - **gte**
       - **not**
-   3. Type the value you want to use for the filter.<br />
-      **Step result**: The model is filtered immediately after you define the first filter.
+   3. Type the value you want to use for the condition.<br />
+      **Step result**: The data in the model is updated immediately based on the condition.
 3. To add a new filter, click **Add a new filter** and repeat the steps above.
-4. To remove a filter, click the **x** button on the right.
-   <img src="https://res.cloudinary.com/prismaio/image/upload/v1658948754/docs/pdp/data-browser-filters-actions-v03_fuewwd.png" width="650px" alt="Data Browser - add and remove apply filters"/>
-5. To remove all filters, click **Clear all**.
-6. If you have too many filters and want to clear the UI real estate, click **Show less** and only the topmost filter remains visible. Click **Show more** to show again the rest of the applied filters.
+4. To clear a filter, click the **x** button on the right.
```

Comment:
Intentional change from "remove" to "clear"?

Reply:
Matching the UI term.

Reply:
From "Clear all"? That is indeed weird.

For me "clear" means removing its setting or values. If you remove an individual one, I would always call that ... "remove".

Reply:
I see what you mean. Switched this back to "remove".